components/centraldashboard/public/components/main-page.js

```diff
@@ -90,7 +91,8 @@ export class MainPage extends PolymerElement {
             buildVersion: {type: String, value: BUILD_VERSION},
             dashVersion: {type: String, value: VERSION},
             inIframe: {type: Boolean, value: false, readOnly: true},
-            _devMode: {type: Boolean, value: DEVMODE},
```

Comment:
Why are we removing this?
I assume we will be hiding off functionality for new releases as well, if they're longer running, or require work with other components (eg. pipelines, notebooks, etc)